### PR TITLE
Fix export-button

### PR DESCRIPTION
**Pull request by jotoeri**
Now with small top-bar i would suppose to make the export button the same height. ;)

grafik

### CODE REVIEW

**Comment by skjnldsv** [comment]
I thought I pushed this already :(

File: src/views/Results.vue [hunk 1]
```diff
@@ -39,7 +39,7 @@
 
 		<header v-if="!noSubmissions">
 			<h2>{{ t('forms', 'Responses for {title}', { title: form.title }) }}</h2>
-			<div>
+			<div id="responseActions">
```
Comment (skjnldsv) [inline src/views/Results.vue:42]: Please use the BEM format and no id :)
```suggestion
			<div class="response_actions">
```

**Comment by jotoeri** [comment]
Based on current master - maybe you wanted on topbar-branch, but it got lost?

File: src/views/Results.vue [hunk 2]
```diff
@@ -39,7 +39,7 @@
...
 				<button id="exportButton" @click="download">
```
Comment (skjnldsv) [inline src/views/Results.vue:43]:
```suggestion
				<button class="response_actions__export" @click="download">
```

File: src/views/Results.vue [hunk 3]
```diff
@@ -234,9 +234,13 @@ h2 {
 	white-space: nowrap;
 }
 
-#exportButton {
-	width: max-content;
-	padding: 13px 16px;
-	margin-left: 16px;
+#responseActions {
```
Comment (skjnldsv) [inline src/views/Results.vue:237]:
```suggestion
.response_actions {
```

File: src/views/Results.vue [hunk 4]
```diff
@@ -234,9 +234,13 @@ h2 {
...
+	display: flex;
+	align-items: center;
+
+	#exportButton {
```
Comment (skjnldsv) [inline src/views/Results.vue:241]:
```suggestion
	&__export {
```

File: src/views/Results.vue (hunk 2 above)
Comment (jotoeri) [reply src/views/Results.vue]: Not my change, but yes, your're right.

**Comment by jotoeri** [comment]
Ah - just added you as co-author now... ;)

**Pull request merged**